pcsx2/GS/GSdx.rc

```diff
@@ -393,10 +393,10 @@ BEGIN
             VALUE "CompanyName", "Gabest"
             VALUE "FileDescription", "GS plugin for ps2 emulators"
             VALUE "FileVersion", "1, 2, 0, 0"
-            VALUE "InternalName", "GSdx.dll"
+            VALUE "InternalName", "GS.dll"
```

Comment:
Can get rid of the entire VERSIONINFO code block.

Reply:
can get rid of the whole file* waiting for the windows port to do so